CLAUDE.md

```diff
@@ -18,13 +18,43 @@
 - **Build**: `pnpm build` (production) or `pnpm build:dev` (development)
 - **Lint**: `pnpm lint` or `pnpm lint:fix` to auto-fix issues
 - **TypeCheck**: `pnpm type-check` or `pnpm check`
+- **Format**: `pnpm prettier --write .` or `pnpm prettier --check .` to check formatting
 - **Test**:
   - All tests: `pnpm test`
   - Single test: `pnpm test -- -t "test name pattern"`
   - E2E tests: `pnpm test-e2e`
   - Coverage: `pnpm test-coverage`
 - **Dev**: `pnpm dev` (runs Next.js on port 2525)
 
+## Git Workflow
+
+### Branch Naming
+
+- **Feature branches**: `feat/description` or `feat/AI-XXXX-description`
```

Comment:
Minor but perhaps the latter should come first to show it's preferred (where a piece of work as an associated ticket).